data/freesurfer_fsaverage/LICENSE

```diff
@@ -0,0 +1,191 @@
+         FreeSurfer Software License Agreement ("Agreement")
```

Comment:
I think the idea to ask the FS mailing list is a good one. Shall I do it, or do you want to ask?

I saw the  word Software in the title, but I consider the second paragraph in the license document, which seems to be saying that they refer by "Software" to both code and data. Did I misunderstood? 

> .... distributing or use of any software and/or data from FreeSurfer (collectively, the "Software")